### ISSUE TITLE
Review DwC mapping

### ISSUE THREAD

**Issue by damianooldoni** [riparias/pov-ias-plants-occurrences#17]
Hi @peterdesmet, @LienReyserhove! The mapping of this dataset is ready. I would like to have your review on the mapping before @LienReyserhove can publish this dataset on GBIF.

**Comment by peterdesmet** [comment]
- [ ] type: should be `Event`
- [ ] license: is CC BY-NC a request by POV?
- [ ] rightsHolder: is `POV` really used as a recognizable acronym for "Provincie Oost-Vlaanderen"? I see it occurring for "Provinciaal Onderwijs Vlaanderen". I would opt to use `PCM` here, as that seems at least used by Provenciaal Centrum voor Milieuonderzoek: https://oost-vlaanderen.be/wonen-en-leven/natuur-en-milieu/provinciaal-centrumvoormilieuonderzoek.html. I would then also change the shortname and repo name.
- [ ] accessRights: these ones are not really applicable here, would drop the term
- [ ] institutionCode: same remark as rightsHolder
- [ ] datasetTitle: is currently really long (`Monitoring of invasive alien species by the Provincial Center of Environmental Research, Province East-Flanders, Belgium`). Would opt for `Monitoring of invasive alien species by PRM in East Flanders, Belgium`
- [ ] recordedBy: could be added with same value as `institutionCode`
- [ ] individualCount: empty for all, drop field
- [ ] list `eventDate` right before countryCode (after occurrence terms)
- [ ] continent: not (no longer) used for rato-occurrences @LienReyserhove should we drop this for all datasets?
- [ ] verbatimLatitude/verbatimLongitude: these are expressed as meters, so round decimal numbers (e.g. `192736.48239376` -> `192736`)
- [ ] scientificName: `Ludwigia peploides montevidensis` should be written as `Ludwigia peploides subsp. montevidensis`
- [ ] taxonRank: would add this term too, before vernacularName. Note that there are `genus`, `species` and `subspecies`

**Comment by damianooldoni** [comment]
> license: is CC BY-NC a request by POV?

Yes

**Comment by damianooldoni** [comment]
Great @peterdesmet! Thanks. I agree and I will update the mapping.

**Comment by damianooldoni** [comment]
> occurrenceID: why the Exo_? Are the data stored like this?

Yes

**Comment by damianooldoni** [comment]
> occurrenceID: note the rows Exo_337 = Exo_338 and Exo_341 = Exo_343. Maybe those duplicate records should be removed in the source data? 

Good catch! I will remove them as first step of the mapping (clening data). The raw csv file comes directly from WFS of province and I would not touch it.

**Comment by damianooldoni** [comment]
> coordinates: a number of records fall outside East Flanders, that makes stateProvince incorrect

You are right. I remove the term.
